docs/Techincal.md

```diff
@@ -17,6 +17,18 @@ This documentation is for Etherpad-next collaborator.
 - [eslint](https://www.eslint.org)
 - [stylelint](https://www.stylelint.io)
 
+## Setup
+
+### Development
+
+```
+docker compose up -d
+docker compose exec app npm i
+# creates db structure without using migrations
+docker compose exec npx prisma db push
+docker compose exec app npm run dev
```

Comment:
```suggestion
docker compose exec app npx turbo dev
```

why not use turborepo ?

Reply:
`npm run dev` calls the script `next dev --turbo` from package.json. Or what are you referring to ?

Reply:
Vercel funrnish two tools call turbo (it's will become one). Turbo repo tools for monorepo + caching. And turbopack (`--turbo`) flag.

If you want to have better understanding of theses awesome tools I recommend you to reed https://www.turbo.build

Reply:
thanks for the link, but I am still not sure what your comment is referring to. My goal is to introduce docker, not to change the build system. I am simply calling your existing package.json script for running the dev server: `npm run dev` and there its using `next dev --turbo` (turbopack). turbo repo tools are not installed yet (I think) and it was not my goal to introduce a different build system now.

Reply:
maybe I get it now. Have you installed on your machine turbo globally? because then this is the reason my its currently missing for me, as its not given as a dependency in the project.

Reply:
look up on this docs it's use npx turbo. And yes I had install it globally and I also use npx 
https://github.com/ether/etherpad-next/blob/main/docs/User.md#how-to-run-etherpad-next

Reply:
ok it now also used turbo. I installed it in the repo as this is recommended, see https://turbo.build/repo/docs/installing#install-per-repository